src/main/java/frc/robot/subsystems/solenoid/SolenoidState.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems.solenoid;
+
+public enum SolenoidState {
+
+	OFF(0),
```

Comment:
Wha5 does off mean

Reply:
not axtivating anything, like not holding and not retracting. when the solenoid is using 0 power